atlp-rwanda/hackers-ec-be
# #187419065 Fixing productStatus type from string to boolean

#### What does this PR do?
Changing product availability status  type from string to boolean

#### Description of Task to be completed?
- Changing product availability status  type from string to boolean
- Making update status an on/off button

#### How should this be manually tested?

Try to access all/single products as buyer and seller and see the results, buyer see only available products

#### What are the relevant pivotal tracker/Trello stories?

#187419065
